02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+                "summary": "Yes, students can be part of a SNAP household if they meet certain criteria, such as being under 18 or over 50, having a disability, or being responsible for a dependent household member."
+            }
+        }
+    },
+    {
+        "id": 2,
+        "question": "The client's son is 20 and is still living with them, but has his own job and buys his own food. The client doesn't want to list him on the application, do they have to? If so, do we need to include the son's income?",
+        "guru_cards": [
+            "Who are mandatory HH members for food stamps?",
+            "How is the income of a dependent counted for SNAP?"
+        ],
+        "recall": 0.5,
+        "extra_cards": 4,
+        "derived_questions": [
+            "Do clients have to list their adult children living with them if they have jobs and buy their own food?",
+            "If they do have to list them, do we need to include the adult child's income?"
+        ],
+        "all_retrieved_cards": {
+            "Can children under 18 years of age ever be certified as a separate household from other adults living in the home?": {
+                "score_sum": 0.8556045602793476,
+                "derived_questions": [
+                    "Do clients have to list their adult children living with them if they have jobs and buy their own food?",
+                    "If they do have to list them, do we need to include the adult child's income?"
+                ],
+                "entire_card": "Can children under 18 years of age ever be certified as a separate household from other adults living in the home?\nAnswer:\nYes, if the adults with whom they are living do not exercise parental control or offer financial support. Clients who are officially emancipated usually fall under this category. If the client not entirely certain or it's close but messy we can error on the side of applying and inform the client the county will help them figure out the household if need be.\nExample: Ms. G is 17 years old. She lives with her aunt. Ms. G works and purchases and prepares her own meals. When questioned about household composition she stated that her aunt does not exercise parental control over her. The case manager certifies Ms. G as a one-person household.\nFood stamps only",
+                "summary": "Children under 18 years of age can be certified as a separate household from other adults living in the home if the adults do not exercise parental control or offer financial support, and clients who are officially emancipated usually fall under this category."
```

Comment:
**Relevance to the question**: 🟠 Potentially relevant
- The original question specifies that the client has a 20yo child. Based on the retrieved Guru card, the summary describes the rules related to children under 18yo being certified as a separate household. The user would need to reason whether to infer that the same rules apply to children over 18yo, or seek a different answer.

**Correctness as a summary of the Guru card**: 🟢 100% correct
- Correctly interprets official emancipation example from the Guru card, improving on [line 24](https://github.com/navapbc/labs-gen-ai-experiments/pull/21/files#r1589750177) above

**Helpfulness**: 🔴 Potentially harmful
- The summary references rules for children under 18, while the question is about a 20yo child
- The summary seems to imply that the 20yo child might _not_ need to be included in the household (inferring from the rules for children under 18), but the correct answer is that they are a mandatory household member